libraries/AP_Frsky_Telem/AP_Frsky_Parameters.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include "AP_Frsky_Parameters.h"
+
+#if HAL_WITH_FRSKY_TELEM_BIDIRECTIONAL
+
+const AP_Param::GroupInfo AP_Frsky_Parameters::var_info[] = {
+    // @Param: UPLINK_ID
+    // @DisplayName: Select frsky passthrough telemetry uplink sensor id
+    // @Description: Change the default uplink sensor id, requires reboot
+    // @Values: -1=Disable, 7 - 26
+    // @User: Advanced
+    AP_GROUPINFO("UPLINK_ID",  1, AP_Frsky_Parameters, _sport_uplink_id, FRSKY_UPLINK_ID_DEFAULT),
+
+    // @Param: DNLINK1_ID
+    // @DisplayName: Select frsky passthrough telemetry additional first downlink sensor id
+    // @Description: Change the default extra downlink sensor id, requires reboot
```

Comment:
One of this or the `DLINK2_ID` descriptions needs fixing.

Reply:
fixed